onnxruntime/test/optimizer/qdq_test_utils.h

```diff
@@ -187,5 +187,27 @@ GetQDQTestCaseFn BuildBinaryOpTestCase(const std::vector<int64_t>& input_shape,
                                                 output_arg);
   };
 }
+
+template <typename InputType, typename OutputType>
+GetQDQTestCaseFn BuildQDQTransposeTestCase(
+    const std::vector<int64_t>& input_shape,
+    const std::vector<int64_t>& perms) {
+    return [input_shape, perms](ModelTestBuilder& builder) {
+      auto* input_arg = builder.MakeInput<InputType>(input_shape, -128, 127);
+      auto* output_arg = builder.MakeOutput();
+
+      // add DQ
+      auto* dq_output = builder.MakeIntermediate();
+      builder.AddDequantizeLinearNode<InputType>(input_arg, .003f, 1, dq_output);
```

Comment:
>1 [](http://example.com/codeflow?start=67&length=1)

Probably a different zero point than the int8 case, such as 
std::numeric_limits<InputType>::max() / 2